packages/api/src/schema/db-schema.yaml

```diff
@@ -995,18 +995,12 @@ components:
             - prod_O9XtHhI6rbTT1B
             - prod_O9XtcfOSMjSD5L
             - prod_O9XuWMU1Up6QKf
-        newStripeProductId:
-          type: string
-          enum:
-            - hacker_1
-            - growth_1
-            - scale_1
-            - pay_as_you_go_1
-            - prod_4
-            - prod_O9XuIjn7EqYRVW
-            - prod_O9XtHhI6rbTT1B
-            - prod_O9XtcfOSMjSD5L
-            - prod_O9XuWMU1Up6QKf
+        toMigrate:
+          type: boolean
+          default: true
+        lastPaymentFailed:
```

Comment:
Is this `lastPaymentFailed` or `lastEmailAboutPaymentFailure`? Might want to add at least a description explaining what really is this field, in case it is not what the name implies (the other one is too verbose I don't like it either haha)

Reply:
Good point! Yeah changing to `lastEmailAboutPaymentFailure`